docs/agents/sematext-agent/releasenotes.md

```diff
@@ -35,6 +35,28 @@ NULL
 
 -->
 
+## Version 3.1.0
+
+Date: August 24, 2023
+
+### New Features
+
+- Varnish Cache metrics monitoring support: With this update, Sematext Agent can now discover and and collect metrics from your Varnish Cache service. Stay tuned for the upcoming release of Varnish Cache monitoring integration in Sematext Cloud.
+- Support has been added for kube-proxy and API server monitoring in Google Kubernetes Engine (GKE) as part of our Kubernetes monitoring integration.
+
+### Improvements
+
+- We've fine-tuned data collection in the Kubernetes monitoring integration, resulting in improved performance and efficiency.
+- Improved discovery for ElasticSearch integration: We have enhanced the ElasticSearch integration by preventing the discovery of `.log` files, which are not supported. Now, Sematext Agent will only discover and support `.json` files for ElasticSearch integration.
+- Enriched Vector diagnostics information for baremetal, Kubernetes, and Docker setups.
+- Introducing unified runtime metrics (goCollector) to provide a comprehensive view of system performance in the Kubernetes monitoring integration. You check all supported runtime metrics in [this wiki](https://sematext.com/docs/integration/kubernetes/#runtime-metrics).
```

Comment:
goCollector? If it's internal is OK, if is public I would change it a bit before we expose

Reply:
This is public, okay will remove `(goCollector)`